src/WebJobs.Extensions.ApiHub/Listener/ApiHubListener.cs

```diff
@@ -86,5 +140,68 @@ private Task OnFileWatcher(IFileItem file, object obj)
                 _poll = null;
             }
         }
+
+        private async Task<string> GetLastPollStatusAsync()
```

Comment:
You need to add some tests for this code. Recommend adding some tests that exercise Get/Set to verify that the blobs are created correctly, etc.